modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js

```diff
@@ -260,26 +305,45 @@ export default function DataAndViewsTab({
 		openFormViewModal(id, defaultLanguageId, updateFormView);
 	};
 
-	const addFormViewButton = (selectFormView) => (
+	const AddButton = (props) => (
 		<ClayTooltipProvider>
 			<Button
 				className="btn btn-monospaced btn-secondary mr-2 nav-btn nav-btn-monospaced"
 				data-tooltip-align="bottom-right"
 				data-tooltip-delay="0"
 				displayType="secondary"
-				onClick={() =>
-					openFormViewModal(
-						dataObject.id,
-						dataObject.defaultLanguageId,
-						selectFormView
-					)
-				}
 				symbol="plus"
-				title={Liferay.Language.get('new-form-view')}
+				{...props}
 			/>
 		</ClayTooltipProvider>
 	);
 
+	const addFormViewButton = (selectFormView) => (
+		<AddButton
+			onClick={() =>
+				openFormViewModal(
+					dataObject.id,
+					dataObject.defaultLanguageId,
+					selectFormView
+				)
+			}
+			title={Liferay.Language.get('new-form-view')}
+		/>
+	);
+
+	const addTableViewButton = (selectTableView) => (
+		<AddButton
+			onClick={() =>
+				openTableViewModal(
+					dataObject.id,
+					dataObject.defaultLanguageId,
+					selectTableView
+				)
+			}
+			title={Liferay.Language.get('table-form-view')}
```

Comment:
The text should be 'new-table-view', right?

Reply:
Nice @naraandrade!

Reply:
Good catch @naraandrade

Reply:
Nice @naraandrade